frontend/src/components/ProductForm/ProductForm.js

```diff
@@ -88,7 +116,8 @@ const ProductForm = ({ match, history }) => {
         <Fields.TextArea onChange={onChange} placeholder={t('product-cru:description')} name="description" value={product.description}/>
         {editId ? null : <Fields.FileUpload onChange={onChange} label="Upload photo" name="_photos" value={product._photos}
                            helpText="JPEG .JPG .PNG (Just these file formats will work)"/>}
-        </>} footer={<Button label={`${t('first-product:next')} →`}/>}
+        </>}
+        footer={ <Button label={editId ? `${t('first-product:edit')} →` : `${t('first-product:add')} →` }/>}
```

Comment:
first-product will always be add, that's the logic of the onboarding process. After you create your company you get to a special view where you can create your very first product and after that you always do it in another view than first-product (which we might also have to add back in later, because I simplified too much)